README.md

```diff
@@ -1,5 +1,79 @@
 # Link Diver
-
+ 
 Link Diver is a chrome extension to speed up understanding the links that can be extracted from web pages, which is a complicated task in modern websites that make heavy use of anchoring and javascript in a sophisticated layout. The main goal is to make it easier for a human to “dive” into web sites just like a web crawler. This should greatly help to efficiently identify patterns in outlinks, problematic URLs, and in general the path that a hypothetical web spider would follow through a website.
-
+ 
 *This is not an officially supported Google product*
+ 
+## Installation Instructions
+ 
+For installation instruction please refer to the latest release which can be found [here](https://github.com/google/link-diver/releases):
+ 
+## User Manual
+ 
+Once you have installed the extension onto chrome you are ready to start diving into web pages! Once on a page that you would link to explore, click on the extension icon in the top right corner of Chrome, or right click anywhere on the page and select the link diver context menu item. A tab will open in a new window with a list of all of the links on the page you were previously on. With the extension now open you can view metadata for each link, filter the links by a variety of options, group the links by various metadata attributes, sort the links, and highlight links that you would like to investigate more thoroughly on the original web page.
+ 
+### Metadata
+ 
+In addition to viewing all of the links available on a page, you can expand each link to see associated metadata about that particular link. To expand a link click on the triangle to the left of the link, or on any other part of the link's box that is not the link itself. You can also expand and collapse each link at once with the 'Expand All' and 'Collapse All' buttons. The following are metadata that will appear upon expanding a link:
+ 
+- **Host** (The host of the web page being linked)
+- **Visible** (Whether or not the link is visible on the original page)
+- **Tag Source** (The type of tag that the link was found under. For example, 'A' for a typical link, 'IMG' for an image, etc.)
+- **Content Type** (The content type of the http response of the request sent to that link)
+- **Status Code** (The status code of the http response)
+ 
+### Fetching HTTP Status Codes
+ 
+On top of displaying each link to you, the Link Diver will send http get requests and display the status code and content type of the response for each link that it collects. Please note that to not overburden the network, these results will come in in batches. Because of this the status code and content type attributes may take a while to appear for every link. If a link has not received a response yet it will simply not display a status code or content type.
+ 
+### Filtering
```

Comment:
Please add a warning saying that this filtering syntax is not meant to be final. We want to update change this in future update.